src/static/css/cf-enhancements.less

```diff
@@ -789,11 +784,13 @@ tbody {
                 .list__icons .list_text();
             }
 
-            &:before {
+            &:after {
                 .cf-icon();
-                .list__icons .list_icon();
-                content: "\e406";
+                content: @cf-icon-download;
                 color: @pacific;
+                //offset to address lack of space between text and icon
```

Comment:
Trivial: Could update to `// Offset to address lack of space between text and icon.`